src/game/World/WorldState.cpp

```diff
@@ -1753,8 +1757,8 @@ void WorldState::StartNewCityAttack(uint32 zoneId)
     if (m_siData.m_pendingPallids.find(zoneId) != m_siData.m_pendingPallids.end())
         return;
 
-    if (!m_siData.m_attackPoints[zoneId].pallidGuid.IsEmpty())
-        return;
+    /*if (!m_siData.m_attackPoints[zoneId].pallidGuid.IsEmpty())
```

Comment:
I'm not sure what this is meant to do, but it was stopping City Invasions from happening, because after the first city invasion pallidGuid is *never* empty